src/CommunityCommons/javasource/communitycommons/StringUtils.java

```diff
@@ -215,25 +216,41 @@ public static String base64EncodeFile(IContext context, FileDocument file) throw
 	}
 
 	public static String stringFromFile(IContext context, FileDocument source) throws IOException
+	{
+		return stringFromFile(context, source, "UTF-8");
+	}
+	
+	public static String stringFromFile(IContext context, FileDocument source, java.lang.String encoding) throws IOException
 	{
 		if (source == null)
 			return null;
+		if (encoding == null || encoding.trim().length() == 0)
+			encoding = "UTF-8";
 		try (
 			InputStream f = Core.getFileDocumentContent(context, source.getMendixObject());
 		) {
-			return IOUtils.toString(f, StandardCharsets.UTF_8);
+			Charset charset = Charset.forName(encoding);
+			return IOUtils.toString(f, charset);
 		}
 	}
 
 	public static void stringToFile(IContext context, String value, FileDocument destination) throws IOException
+	{
+		stringToFile(context, value, destination, "UTF-8");
+	}
+	
+	public static void stringToFile(IContext context, String value, FileDocument destination, java.lang.String encoding) throws IOException
 	{
 		if (destination == null)
 			throw new IllegalArgumentException("Destination file is null");
 		if (value == null)
 			throw new IllegalArgumentException("Value to write is null");
+		if (encoding == null || encoding.trim().length() == 0)
+			encoding = "UTF-8";
+		Charset charset = Charset.forName(encoding);
```

Comment:
These three lines become irrelevant if you take a Charset parameter instead of a String.